### PR TITLE
setting the right tag: service-bus-messaging

### PR DESCRIPTION
**Pull request by spelluru**

### CODE REVIEW

**Comment by spelluru** [comment]
#sign-off

File: docs-ref-services/latest/servicebus-readme.md [hunk 1]
```diff
@@ -495,30 +494,30 @@ For more information see the [Code of Conduct FAQ](https://opensource.microsoft.
...
 
 <!-- LINKS -->
-[azure_cli]: https://docs.microsoft.com/cli/azure
+[azure_cli]: /cli/azure
 [api_docs]: https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/latest/index.html
-[product_docs]: https://docs.microsoft.com/azure/service-bus-messaging/
+[product_docs]: /azure/service-bus-messaging/
 [azure_portal]: https://portal.azure.com
 [azure_sub]: https://azure.microsoft.com/free/
-[cloud_shell]: https://docs.microsoft.com/azure/cloud-shell/overview
```
Comment (scbedd) [inline docs-ref-services/latest/servicebus-readme.md:503]: This readme is sourced from the _actual_ readme in the docs repo.

I will accept these changes, but understand that the next time this package releases a GA version of the package, these updates will be flattened.

File: docs-ref-services/latest/servicebus-readme.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
+description: Azure Service Bus client library for Python
 keywords: Azure, python, SDK, API, azure-servicebus, servicebusservicebus
 author: maggiepint
 ms.author: magpint
 ms.date: 06/08/2021
 ms.topic: article
-ms.prod: azure
-ms.technology: azure
 ms.devlang: python
-ms.service: servicebusservicebus
```
Comment (scbedd) [inline docs-ref-services/latest/servicebus-readme.md:11]: This is good to know. Given that this was placed by automation, I'll track this back and get it fixed.

**Review by scbedd — APPROVED** [review]
Your changes will be automatically flattened here by the next automated release. I'm tracking the source of `ms.service` and will see what I can do to fix that.

The rest will unfortunately be flattened on the next release (given they are sourced directly from real readmes @ release time).

**Pull request closed without merging**